napari/_qt/qt_main_window.py

```diff
@@ -187,6 +197,8 @@ def add_dock_widget(
             If provided, `widget` will be added as QDockWidget's main widget
         allowed_areas : Qt.DockWidgetArea, optional
             Areas, relative to main window, that the new dock is allowed to go.
+        name : str, optional
+            Name of dock widget to appear in window menu.
```

Comment:
don't know how to comment on non-PR lines... but as long as we have a names argument (which I like!), you could name the dockwidget on line 212:
```python
dock_widget = QDockWidget(str(name or ''), self._qt_window)
```
and then tabs (if visible) will have names: ![Screen Shot 2019-11-20 at 1 07 59 PM](https://user-images.githubusercontent.com/1609449/69265207-cd902f80-0b96-11ea-9332-98972eca3134.png)

adding a name to the DockWidget, however, will then show the word "console" in the title bar ... which I think is unattractive (actually, it's even worse than this image because it's off-centered):

![Screen Shot 2019-11-20 at 1 13 58 PM](https://user-images.githubusercontent.com/1609449/69265667-a25a1000-0b97-11ea-9be7-193923f87543.png)

so to hide it, you can modify `stylesheet.qss` as follows:

```css
QDockWidget::title {
  background-color: {{ background }};
  text-align: center; /* otherwise it overlaps with the buttons */
  margin-top:-50px
}
```

Reply:
great idea, done

Reply:
> don't know how to comment on non-PR lines
me neither - would be a good feature request for GitHub if it isn't possible!!